sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -4923,8 +4923,12 @@ def sdc_pandas_series_combine(self, other, func, fill_value=None):
 
     Limitations
     -----------
-    - Only supports the case when data in series of the same type.
-    - With the default fill_value parameter value, the type of the resulting series will be float.
+    - Resulting series dtype may be wider than in pandas due to
+      type-stability requirements and depends on fill_value dtype
+      and result of series indexes alignment.
+    - Indixes should be strictly ascending, as inside the function
+      they are sorted in ascending order and the answer becomes
+      different from the result of the pandas.
```

Comment:
@Rubtsowa What? This sounds like a bug...

Reply:
This 'bug' in function **sdc_join_series_indexes**

Reply:
@Rubtsowa Then it should be fixed (please create a JIRA case with a reproducer) before this is merged. Adapting to a bug is no way. @AlexanderKalistratov correct?